keyboards/fluorite/keymaps/default/keymap.c

```diff
@@ -0,0 +1,224 @@
+/* Copyright 2019 Mafuyu Ihotsuno
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+enum layer_names {
+    _QWERTY,
+    _LOWER,
+    _RAISE,
+    _CUSTOM1,
+    _CUSTOM2,
+    _ADJUST
+};
+
+// Defines the keycodes used by our macros in process_record_user
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  CUSTOM1,
+  CUSTOM2,
+  ADJUST,
```

Comment:
These three keycodes (and the `ADJUST` case in `process_record_user()`) don't seem to be used.

Reply:
I corrected the points that were pointed out.